app/viewmodels/CheckYourAnswersHelper.scala

```diff
@@ -59,6 +60,74 @@ trait CheckYourAnswersHelper extends ImplicitDateFormatter with SummaryListRowHe
       )
     }
 
+  def answerFullame(page: QuestionPage[Name],
+                    changeLinkCall: Call) : Option[SummaryListRow] =
+    userAnswers.get(page) map { ans =>
+      summaryListRow(
+        label = messages(s"$page.checkYourAnswersLabel"),
+        value = ans.getFullName,
+        visuallyHiddenText = Some(messages(s"$page.checkYourAnswersLabel")),
+        changeLinkCall -> messages("site.edit")
+      )
+    }
+
+  def answerPrefix[A](page: QuestionPage[A],
+                changeLinkCall: Call,
+                answerIsMsgKey: Boolean = false,
```

Comment:
as we are passing answerPrefix to this method, answerIsMsgKey is redundant can we remove this and if condition below  

Reply:
answerIsMsgKey is required here because, we display answer from messages for Position rows not from database

Reply:
my question was related to below condition
 if (answerIsMsgKey) messages(s"$messagePrefix.$ans") else ans
is it still valid? as we are passing message kesy and  it always used from messages